joplin/static/css/preview.css

```diff
@@ -42,7 +42,7 @@
 .thumbnail,
 .thumbnail iframe {
   width: 100%;
-  height: 72vh;
+  height: calc(100vh - 242px);
```

Comment:
👏

Reply:
just double checked caniuse.com and we can use `calc` back to IE9 with partial support. Wondering if we should get some usage stats or set a support standard for the CMS. We went through that process with the resident site, but the data would be different for city admins than residents. Probably for the worse 😞